src/sentry/models/groupenvironment.py

```diff
@@ -11,6 +14,7 @@ class GroupEnvironment(Model):
     group_id = BoundedPositiveIntegerField()
     environment_id = BoundedPositiveIntegerField()
     first_release_id = BoundedPositiveIntegerField(null=True)
+    first_seen = DateTimeField(default=timezone.now, db_index=True, null=True)
```

Comment:
i removed the `default` from the migration, so this default will only apply at the application level and won't affect the migration

Reply:
Will adding a default lock the table while all the rows are rewritten? There are quite a few rows in production.

Reply:
Yes.

Reply:
The default won't rewrite the rows if the column is nullable, afaik?

Reply:
It would. If the default is set at the database level, it'll always be applied to all rows, regardless if the column is nullable or not.

Reply:
Interesting, so it looks like if you create the column first, and then alter it to have a default then the default will only apply to new rows, but if you include it in the initial definition it'll apply it to existing. Good to know!